molo/core/tests/test_utils.py

```diff
@@ -0,0 +1,13 @@
+from django.test import TestCase
+from molo.core import utils
+
+
+class TestUtils(TestCase):
+
+    def test_get_locale_code(self):
+        assert utils.get_locale_code() == 'en'
```

Comment:
While doing some digging earlier, I found out that the `py.test` community likes using `assert` and it apparently handles it differently too.
The error messages are just as useful.
![screenshot from 2015-06-04 14 00 34](https://cloud.githubusercontent.com/assets/623417/7983577/1ffce6c6-0ac2-11e5-8de7-6e8475ae99b5.png)